com.unity.netcode.adapter.utp/Tests/Runtime/TransportTests.cs

```diff
@@ -1,3 +1,5 @@
+// todo @simon-lemay-unity: un-guard/re-enable after validating UTP on consoles
```

Comment:
This was required to get tests passing on v1.1.0, approved by Simon.

Reply:
Might recommend creating a tech debt jira and including it in the comment. In my experience these types of todos have a habit of getting forgotten

Reply:
The related task in JIRA is MTT-1904. It's in our current sprint so hopefully these todo's should be addressed very soon.

Reply:
Ah cool thanks for the context, glad you're already on top of it!